Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -19,22 +19,23 @@ struct IntroViewControllerUX {
+    static let CardTitleWelcome = NSLocalizedString("Thanks for choosing Firefox!", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
+    static let CardTitleSearch = NSLocalizedString("Your search, your way", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
     
-    static let CardTitlePrivate = NSLocalizedString("Get stealthy", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
+    static let CardTitlePrivate = NSLocalizedString("Browse like no one's watching", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
     static let CardTitleMail = NSLocalizedString("You've got mail... options", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
-    static let CardTitleSync = NSLocalizedString("Sync your devices", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
+    static let CardTitleSync = NSLocalizedString("Pick up where you left off", tableName: "Intro", comment: "Title for one of the panels in the First Run tour.")
     
-    static let CardTextWelcome = NSLocalizedString("Swipe to take a quick look at a few features we hope you'll enjoy.", tableName: "Intro", comment: "Description for the 'Welcome' panel in the First Run tour.")
-    static let CardTextSearch = NSLocalizedString("There are a lot of search engines out there, so select your favorite one and find what you need.", tableName: "Intro", comment: "Description for the 'Favorite Search Engine' panel in the First Run tour.")
-    static let CardTextPrivate = NSLocalizedString("Tap the mask icon to slip into Private Browsing mode.", tableName: "Intro", comment: "Description for the 'Private Browsing' panel in the First Run tour.")
-    static let CardTextMail = NSLocalizedString("Have another mail application installed? Use it with Firefox.", tableName: "Intro", comment: "Description for the 'Mail' panel in the First Run tour.")
+    static let CardTextWelcome = NSLocalizedString("A modern mobile browser from Mozilla, the non-profit committed to a free and open web.", tableName: "Intro", comment: "Description for the 'Welcome' panel in the First Run tour.")
+    static let CardTextSearch = NSLocalizedString("Searching for something different? Choose another default search engine (or add your own) in Settings.", tableName: "Intro", comment: "Description for the 'Favorite Search Engine' panel in the First Run tour.")
+    static let CardTextPrivate = NSLocalizedString("Tap the mask icon to slip into \nPrivate Browsing mode.", tableName: "Intro", comment: "Description for the 'Private Browsing' panel in the First Run tour.")
+    static let CardTextMail = NSLocalizedString("Use any email app - not just \nMail - with Firefox.", tableName: "Intro", comment: "Description for the 'Mail' panel in the First Run tour.")
```

Comment:
Please remove the `\n`s from all strings. That will not work out well with dynamic font sizes and different width devices.